src/web/handlers/GestureHandler.ts

```diff
@@ -751,60 +686,130 @@ export default abstract class GestureHandler implements IGestureHandler {
     this.delegate.destroy(this.config);
   }
 
-  //
-  // Getters and setters
-  //
+  public getTrackedPointersID(): number[] {
+    return this.pointerTracker.getTrackedPointersID();
+  }
 
-  public getTag(): number {
-    return this.handlerTag;
+  private isFinished(): boolean {
+    return (
+      this.state === State.END ||
+      this.state === State.FAILED ||
+      this.state === State.CANCELLED
+    );
   }
 
-  public setTag(tag: number): void {
-    this.handlerTag = tag;
+  public get lastSentState() {
+    return this._lastSentState;
+  }
+  public set lastSentState(state: State | null) {
+    this._lastSentState = state;
   }
 
-  public getConfig() {
-    return this.config;
+  public get state() {
+    return this._state;
+  }
+  public set state(state: State) {
+    this._state = state;
   }
```

Comment:
Previously the `currentState` was `protected`, and was only used internally. 
Now the `state` is exposed as its getters are `public`.
Same for `tracker`, `hasCustomActivationCriteria`, and multiple other internal variables which were `private`/`protected`, and not exposed by any public getters or setters before your changes.

Reply:
Well, yes and no. Even though those modifiers are public, they are not available to end user. At least doing something like:

```jsx
const pan = Gesture.Pan()
pan.state = State.ACTIVE
```

won't work as the user operates on `Gesture` type.

I've also thought about removing setters/getters that are not used, but I'm not sure about that.

Either way, we can change access modifiers to match previous state of this implementation, but I don't think it makes much sense to create `private` getters and setters. CC @j-piasecki